### DISCUSSION TITLE
Actual working mac/linux batches.

### DISCUSSION THREAD

**Discussion by aseichter2007** [aseichter2007/ClipboardConqueror#4] [Announcements]
I figured I should push this out as a release. Grab the source, read the readme, and you're good to go

install first 

your choice of run. 

Conquer any task. 

<hr /><em>This discussion was created from the release <a href='https://github.com/aseichter2007/ClipboardConqueror/releases/tag/0.9999'>Actual working mac/linux batches. </a>.</em>

**Comment by teraton** [comment]
something is still wrong with this. I now have koboldcpp installed, I have dependencies installed and the app is running in nodemon, but when I copy something to clipboard nothing happens on kobold or CC terminals where im monitoring. so something on Mac is not running. Kobold runs through the web interface, the problem is somewhere in CC or in how CC connects to kobold.

**Reply by aseichter2007** [reply]
Update and try the new mac .sh, please let me know if it works. My expectation was the clipboard event handler, so the changes should give file execution permission to the correct dependency.

**Reply by teraton** [reply]
Yep, seems to work now! thank you, now to figure out how to disable the additional questions generated somewhere int eh system prompt to get only one answer without the generated extra steps ;)

**Reply by aseichter2007** [reply]
the kobold console shows your entire prompt without any dinking around, it's why I prefer it.

If you mean the captain clip persona, he has instructions to think step by step. That's probably what you don't like.

You can change pretty much everything in setup.js, either quickly at the top(persona: "yourChoice"), or by rewriting default at the bottom to match your taste. ctrl + f "Ident"

check out the [default agents](https://github.com/aseichter2007/ClipboardConqueror/blob/main/agents.md) to see quick descriptions of the prompts I have set up, or read them in their entirety in setup.js toward the bottom.

Or write your own and use |||set|. 

If you want to be sure to keep your agents through updates, enable endpoints.writeFiles in setup.js, but as I keep adding settings and updating prompt formats, consider disabling writeFiles and deleting all but 0-identities.json.

I'm in the middle of adding automatic writing of 0identites.json when you do your first |||agent:file|
